spacy/pipeline/pipe.pyx

```diff
@@ -269,81 +78,24 @@ cdef class Pipe:
         """
         return {}
 
-    def to_bytes(self, *, exclude=tuple()):
-        """Serialize the pipe to a bytestring.
-
-        exclude (Iterable[str]): String names of serialization fields to exclude.
-        RETURNS (bytes): The serialized object.
-
-        DOCS: https://nightly.spacy.io/api/pipe#to_bytes
-        """
-        serialize = {}
-        serialize["cfg"] = lambda: srsly.json_dumps(self.cfg)
-        serialize["model"] = self.model.to_bytes
-        if hasattr(self, "vocab"):
-            serialize["vocab"] = self.vocab.to_bytes
-        return util.to_bytes(serialize, exclude)
-
-    def from_bytes(self, bytes_data, *, exclude=tuple()):
-        """Load the pipe from a bytestring.
-
-        exclude (Iterable[str]): String names of serialization fields to exclude.
-        RETURNS (Pipe): The loaded object.
+    def is_trainable(self) -> bool:
```

Comment:
Why did we make this a method rather than a property?

Reply:
Presumably because `is_resizable` is also a method (not sure why, though?)

Reply:
I think they should both be properties.

Reply:
Yep, agreed!